TwitchChatSharp/TwitchChatSharp.csproj

```diff
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>TwitchChatSharp</RootNamespace>
     <AssemblyName>TwitchChatSharp</AssemblyName>
-    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.7.1</TargetFrameworkVersion>
```

Comment:
long term the target should be a .net standard version, not framework, so it can be used with either core or framework, but I'm not going to ask you to make that change

Reply:
I don't have core installed and just updated it to what my bot is using.